resources/gemini-1b/projects.tf

```diff
@@ -1,13 +1,14 @@
-resource "digitalocean_project" "gemini-1b" {
-  name        = "gemini-1b"
+resource "digitalocean_project" "tf_refactoring" {
+  name        = "tf_refactoring"
```

Comment:
I don't see such project in DO, but I do see empty `gemini-1b` project. If this doesn't correspond to DO state what else doesn't?

Reply:
tf_refactoring was used to test tf cloud and refactor current gemini-1b setup. I can switch it back to gemini

Reply:
I mean if we follow infrastructure-as-code, then code should be identical to infrastructure and vice versa. It is not "I can" kind of situation, it feels like this repo contains some random terraform code that isn't representative of our actual infrastructure. If that is the case we're in a really bad shape because neither `main` nor this PR match the actual infrastructure.

This should be avoided as much as possible, please make sure this PR contains code that **actually** corresponds to the infrastructure we have or else I don't see the point of using terraform and have this repo at all.